iron-image.html

```diff
@@ -224,70 +268,98 @@
           type: Number,
           value: null
         },
-
-        _placeholderBackgroundUrl: {
-          type: String,
-          computed: '_computePlaceholderBackgroundUrl(preload,placeholder)',
-          observer: '_placeholderBackgroundUrlChanged'
-        },
-
-        requiresPreload: {
-          type: Boolean,
-          computed: '_computeRequiresPreload(preload,loaded)'
-        },
-
-        canLoad: {
-          type: Boolean,
-          computed: '_computeCanLoad(preventLoad, src)'
-        }
       },
 
       observers: [
         '_transformChanged(sizing, position)',
-        '_loadBehaviorChanged(canLoad, preload, loaded)',
-        '_loadStateChanged(src, preload, loaded)',
+        // Duplicate observers are needed to work around a bug where Polymer
+        // does not update multi-parameter observers when one parameter
+        // changes to undefined.
```

Comment:
The implementation would definitely read more cleanly as a computed value that acts on `alt` and `src` changes.